packages/framework/src/Command/DispatchRecalculationMessageCommand.php

```
@@ -143,9 +143,10 @@ protected function executeIds(
             return Command::FAILURE;
         }
 
-        $dispatchedProductIds = $this->productRecalculationDispatcher->dispatchProductIds($productIds, $priority, $scopes);
```

If I dispatch a message with a non-existing scope, the command behaves like nothing is wrong, but then just silently fails and moves the message to the failed queue. 
It would be nice to check scopes before and even check if the code warns programmer if he tries to export a non-existing scope programmatically (e.g., with `dispatchProductIds` method)

---

Yes, scopes are now verified before dispatching products batch